tracee-ebpf/tracee/process_tree.go

```diff
@@ -155,6 +155,8 @@ func NewProcessTree() (*ProcessTree, error) {
 		}
 		taskDir, err := os.Open(fmt.Sprintf("/proc/%d/task", pid))
 		processTasks, err := taskDir.Readdirnames(-1)
+
+		// if it failed to parse the task dir we continue to avoid failing tracee
```

Comment:
I would put such a comment above the continue statement below, saying that:
// process might have exited - ignore this task